src/feature/featurelist.h

```diff
@@ -38,14 +41,21 @@ FEATURE(annualUpgrade,         // Feature ID
         FeatureCallback_true,  // Can be flipped on
         FeatureCallback_true,  // Can be flipped off
         QStringList(),         // feature dependencies
-        FeatureCallback_annualUpgrade)
+        enableForPlatform({
+            .windows = true,
+            .macos = true,
+            .linux = true,
```

Comment:
To match prior, it should have WASM here.

Reply:
thanks! fixed.